wpilibj/generate.bzl

```diff
@@ -0,0 +1,29 @@
+def __generate_wpilibj_impl(ctx):
```

Comment:
I don't disagree with a rule on principle, but I feel like I've probably seen something simple in aspect's lib code which would make this simpler.  This is essentially a genrule.